sites/docs/src/content/docs/usage/quick_start/installation.md

```diff
@@ -0,0 +1,216 @@
+
+```bash
+curl -s https://get.sdkman.io | bash
+```
+
+2. Open a new terminal.
+
+3. Install Java:
+
+   ```bash
+   sdk install java 17.0.10-tem
+   ```
+
+4. Confirm that Java is installed correctly:
+
+   ```bash
+   java -version
+   ```
+
+### Install Nextflow
+
+Nextflow is distributed as a self-installing package. It can be installed using a few easy to follow steps:
+
+1. Install Nextflow:
+
+   ```bash
+   curl -s https://get.nextflow.io | bash
+   ```
+
+This will create the `nextflow` executable in the current directory.
+
+2. Make Nextflow executable:
+
+   ```
+   chmod +x nextflow
+   ```
+
+3. Move `nextflow` to an executable path. For example:
+
+   ```bash
+   mkdir -p $HOME/.local/bin/
+   mv nextflow $HOME/.local/bin/
+   ```
+
+   Ensure the directory `$HOME/.local/bin/` is included in your `PATH` variable. Execute `export PATH="$PATH:$HOME/.local/bin"` to add this directory to `PATH` temporarily . Add this command to your shell configuration file (e.g., `~/.bashrc` or `~/.zshrc`) to add this directory to `PATH` permanently.
+
+   :::warning
+   Nextflow will update its executable during the self update process and it should not be placed in a directory with restricted permissions.
+   :::
+
+4. Confirm that Nextflow is installed correctly:
+
+   ```bash
+   nextflow info
+   ```
+
+Alternatively, Nextflow can be installed using [Bioconda](https://bioconda.github.io/):
+
+1. Add conda channels:
+
+   ```bash
+   conda config --add channels bioconda
+   conda config --add channels conda-forge
+   ```
+
+   :::warning
+   Conda channels are the locations where packages are stored. If the channels are not configured accurately your dependencies may be installed from
+   the wrong channels and be incompatible.
+   :::
+
+2. Create and activate a dedicated Nextflow conda environment.
+
+   ```bash
+   conda create --name env_nf nextflow
+   conda activate env_nf
+   ```
+
+   :::note
+   To deactivate the `env_nf` conda environment you can use the `conda deactivate` command.
+   :::
+
+### Update Nextflow
+
+You can update to the latest version of Nextflow using the self update command:
+
+```bash
+nextflow self-update
+```
+
+Specific versions of Nextflow can be installed using the `NXF_VER` environment variable.
+Nextflow will automatically install the specified version at the start of the next command.
+
+```bash
+export NXF_VER=23.10.1
+nextflow run hello-world
+```
+
+:::note
+Set the `NXF_VER` variable in your `.bashrc` or `.zshrc` to fix your version of Nextflow for all sessions.
+:::
+
+You can also temporarily switch to a specific version of Nextflow with the `NXF_VER` environment variable. For example:
+
+```bash
+NXF_VER=23.10.0 nextflow run hello
+```
+
+### Edge releases
+
+A stable version of Nextflow is released every six months, in the 4th and 10th month of each year. Additionally, an edge version is released on a monthly basis. The edge releases can be used to access the latest updates and experimental features.
```

Comment:
```suggestion
A stable version of Nextflow is released every six months, in the 4th and 10th month of each year. Additionally, an edge version is released on a monthly basis. The edge releases can be used to access the latest updates and experimental features. Keep in mind any changes or features in these 'bleeding-edge' releases are not stable and may change in future versions.
```

Reply:
I'll drop this down into a note to make it stand out